packages/ozone/src/mod-service/views.ts

```diff
@@ -163,6 +163,12 @@ export class ModerationViews {
       eventView.event.sticky = true
     }
 
+    if (event.action === 'com.atproto.admin.defs#modEventTag') {
+      eventView.event.add = event.meta?.addedTags?.toString().split(' ') || []
```

Comment:
A common issue with space-separated strings is that we need a little bit more logic than a `split(' ')`.  If the string is empty `''` then the split gives you `['']`, when you probably want `[]`.

Reply:
moved to dedicated jsonb columns as you recommended above so we won't have to worry about the string related edge cases.